External/FEXCore/Source/Interface/Core/Interpreter/ALUOps.cpp

```diff
@@ -113,6 +113,22 @@ DEF_OP(Neg) {
   }
 }
 
+DEF_OP(Abs) {
+  auto Op = IROp->C<IR::IROp_Abs>();
+  const uint8_t OpSize = IROp->Size;
+
+  const uint64_t Src = *GetSrc<int64_t*>(Data->SSAData, Op->Src);
```

Comment:
Is the implicit conversion to `uint64_t` from `int64_t` intentional here?

Reply:
Unintentional, Made sure it didn't need to cast there.